src/app/getting-started/services/provider.service.ts

```diff
@@ -31,8 +36,42 @@ export class ProviderService {
    *
    * @param redirect URL to be redirected to after successful account linking
    */
-  linkGitHub(redirect: string): void {
-    this.link("github", redirect);
+  linkGitHub(redirect: string): void{
+    let headers = new Headers();
+    let githubLinkURL = this.linkUrl + `redirect=` + redirect ;
```

Comment:
There is no reason to use backticks here - single quotes are sufficient if you aren't going to use string interpolation.